src/app/public/modules/data-manager/models/data-manager-column-picker-option.ts

```diff
@@ -11,6 +11,10 @@ export interface SkyDataManagerColumnPickerOption {
    * The description text rendered beneath the column label in the column picker.
    */
   description?: string;
+  /**
+   * Initially hide the column when it is added to the grid unless given in the view state's `displayedColumnIds`
```

Comment:
How do sticky settings impact this boolean? Should we expand on that since this is part of the public API?

Reply:
Updated. Let me know if you still have questions.